Gradle 8 -> Namespace not specified

With gradle 8 there is a new mandatory parameter called "namespace". If I try to incorporate account_picker in a Gradle 8 app I get the error below.

```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':account_picker'.
> Could not create an instance of type com.android.build.api.variant.impl.LibraryVariantBuilderImpl.
   > Namespace not specified. Specify a namespace in the module's build file: /home/xxx/.puro/shared/pub_cache/hosted/pub.dev/account_picker-2.1.0/android/build.gradle. See https://d.android.com/r/tools/upgrade-assistant/set-namespace for information about setting the namespace.

     If you've specified the package attribute in the source AndroidManifest.xml, you can use the AGP Upgrade Assistant to migrate to the namespace value in the build file. Refer to https://d.android.com/r/tools/upgrade-assistant/agp-upgrade-assistant for general information about using the AGP Upgrade Assistant.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

BUILD FAILED in 1s
Error: Gradle task assembleDebug failed with exit code 1

```

The fix is quite simple and backward compatible (in gradle v7 that parameter was optional), but still would require you to release a new verison of course:
https://d.android.com/r/tools/upgrade-assistant/set-namespace


Thank you very much in advance for your work and this plugin!

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -22,6 +22,9 @@ rootProject.allprojects {
 apply plugin: 'com.android.library'
 
 android {
+    if (project.android.hasProperty("namespace")) {
+        namespace 'com.tolotra.account_picker'
+    }
     compileSdkVersion 28
 
     defaultConfig {

